### DISCUSSION TITLE
Update won't update

### DISCUSSION THREAD

**Discussion by groovebak** [brianpetro/obsidian-smart-connections#247] [General]
Hi. I tried updating to 1.6.26 several times (I'm currently on 1.6.25), but it won't let me for some reason. Should I uninstall then reinstall? Will it have to re-index all my notes again?  Thank you! Love this plugin.

**Comment by brianpetro** [comment]
Sorry for the confusion! I just pushed a fix and it should now successfully update.

Thanks for bringing this to my attention
Brian 🌴
